server/src/main/java/io/druid/segment/loading/OmniSegmentLoader.java

```diff
@@ -52,12 +54,14 @@
   public OmniSegmentLoader(
```

Comment:
I'll re-look at it, but I think OmniSegmentLoader still provides some functionality. I'll get a writeup in a bit on if it is still needed.